binstar_client/utils/projects/test/test_filters.py

```diff
@@ -0,0 +1,96 @@
+import unittest
+try:
+    from unittest import mock
+except ImportError:
+    import mock
+from binstar_client.utils.test.utils import example_path
+from binstar_client.utils.projects.filters import (VCSFilter,
```

Comment:
one of two things could be better here:

`from binstar_client.utils.projectsimport filters` then use `filters.VCSFilter`, as stated in best practices from david beasley, rafael de barros and openstack best practices.
or flatten out the imports, which makes it better to remove/project finds.
like this: 
`from binstar_client.utils.projects.filters import VCSFilter`
`from binstar_client.utils.projects.filters import LargeFilesFilter`
...